utilities/tools/helper/ConvertTo-OrderedHashtable.ps1

```diff
@@ -87,13 +87,13 @@ function ConvertTo-OrderedHashtable {
                 }
 
                 # Case: Array of objects
-                $hashTableElements = $JSONObject[$currentLevelKey] | Where-Object { $_.GetType().Name -eq 'Hashtable' }
+                $hashTableElements = $JSONObject[$currentLevelKey] | Where-Object { $_.GetType().Name -in @('Hashtable', 'OrderedHashtable') }
```

Comment:
Does both script changes need to be part of this PR?

Reply:
the one for the removal - yes. That one is specific to this modules test cases.

The other one (which was accidently merged) I tried to undo, but it continues to show a non-change - so yeah, I guess that stays.